internal/cluster/transition.go

```diff
@@ -153,14 +156,35 @@ func (th *transitionHandler) PostCompleteInstallation(sw stateswitch.StateSwitch
 		return errors.New("PostCompleteInstallation invalid argument")
 	}
 
-	if cluster, err := params.clusterAPI.CompleteInstallation(params.ctx, params.db, sCluster.cluster, true, statusInfoInstalled); err != nil {
+	if cluster, err := params.clusterAPI.CompleteInstallation(params.ctx, params.db, sCluster.cluster,
+		true, createClusterCompletionStatusInfo(sCluster.cluster)); err != nil {
 		return err
 	} else {
 		sCluster.cluster = cluster
 		return nil
 	}
 }
 
+func createClusterCompletionStatusInfo(cluster *common.Cluster) string {
+	_, statuses := getClusterMonitoringOperatorsStatus(cluster)
+
+	// Cluster status info is installed if no failed OLM operators
+	if countOperators(statuses[models.OperatorTypeOlm]) == 0 ||
+		len(statuses[models.OperatorTypeOlm][models.OperatorStatusFailed]) == 0 {
+		return statusInfoInstalled
+	}
+
+	statusInfo := StatusInfoDegraded
+	for _, status := range []models.OperatorStatus{models.OperatorStatusAvailable, models.OperatorStatusFailed} {
```

Comment:
i think we can add only failed ones no?

Reply:
Clearer information of the cluster operators

Reply:
I agree with @tsorya .  This string might get long, especially when we start reporting failed workers.